components/spotlight/SpotlightDetail.vue

```diff
@@ -24,14 +25,15 @@ type NftSimpleView = {
 }
 
 @Component({ components })
-export default class SpotlightDetail extends Vue {
+export default class SpotlightDetail extends mixins(PrefixMixin) {
   @Prop(String) public account!: string
   protected nfts: NftSimpleView[] = []
   protected isLoading = true
 
   protected async fetchNFT(account: string) {
     const nfts = await this.$apollo.query({
       query: nftSimpleListByAccount,
+      client: this.urlPrefix,
```

Comment:
spotlight has nothing to do with prefixmixin

Reply:
@vikiival it throw this without client
<img width="761" alt="Capture d’écran 2022-01-19 à 8 21 58 AM" src="https://user-images.githubusercontent.com/9987732/150083283-07684f39-e01c-457b-82e0-f3e35db95fde.png">

Reply:
```suggestion
      client: 'subsquid',
```

Reply:
not subsquid, rmrk

Reply:
pushed some changes